### ISSUE TITLE
The new version of Clean Flash now only installs the legacy Active X plugin on Windows 7 and older.

### ISSUE THREAD

**Issue by nima-amin** [CleanFlash/installer#21]
Means exactly what?
That if you use Windows 10 and need the active x version to run a screensaver or a programme like SWF Player or programme element like the Irfavn View FP Plugin it is no longer possible to update?

**Comment by darktohka** [comment]
You can still update, there will be no problem.

The issue was about unintended behaviour that would install the wrong version of the ActiveX plugin on newer systems, as a special build exists for every Windows 8+ system.

Since this issue has been resolved, now, the correct version will be installed on BOTH Windows 7 and older AND Windows 8 and newer.

So, feel free to update!

**Comment by darktohka** [comment]
In layman's terms: "Videos that didn't work in Flash on older releases using Windows 8+ will now work correctly"
